### DISCUSSION TITLE
migration problem

### DISCUSSION THREAD

**Discussion by elenaaralla** [serenity-is/Serenity#6812] [Help]
Hi all,

maybe someone get the same problem

I'm trying to use _FluentMigrator.DotNet.Cli_ to generate migration file, but I got this strange error running this command:
**dotnet-fm list migrations -p sqlserver -a "my assembly path" --allowDirtyAssemblies**:

```
dotnet-fm list migrations -p sqlserver -a "D:\WORKING\PecDownloaderAdmin\PecDownloaderAdmin.Web\bin\Debug\net6.0\PecDownloaderAdmin.Web.dll" --allowDirtyAssemblies
// Wrong assembly binding redirects for System.Runtime, loading it from disk anyway. C:\Program Files\dotnet\shared\Microsoft.NETCore.App\5.0.4\System.Runtime.dll
dotnet-fm : Unhandled exception. System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.AspNetCore.Mvc.ViewFeatures, Version=6.0.0.0, 
```

I searched on google with no success :(

I'm using FluentMigrator.DotNet.Cli to generate TSQL script for production environment.

Thank you,
Elena.

**Comment by VictorTomaili** [comment]
You need to ask in FluentMigrator repository.

**Reply by elenaaralla** [reply]
Already done; no answer at the moment :)

Thank you
